harvard-lts/CURIOSity
# Merge from 1-24-2022

**Merge from 1-24-2022*
* * *


# What does this Pull Request do?
Gets 2 PRs into our main:

- fix the timestamps view
- hides guests in drop down

# How should this be tested?

You can pull and build this branch and view timestamps in the dashboard to check the timestamps PR. Follow the instructions found here to test the guests hiding:
https://github.com/scientist-softserv/harvard-curiosity/pull/37

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests?
- integration tests?

# Interested parties
Tag (@ mention) interested parties
